content/pages/the-missions/cluster-workshop.md

```diff
@@ -0,0 +1,90 @@
+at BU in Boston, Massachusetts. 
+
+Since its launch in 2009, the Kepler telescope has been providing high precision photometric time series on thousands of 
+targets, from exoplanet hosts to active dwarf stars. With its focus on the ecliptic plane, the K2 Mission has further 
+expanded science opportunities, especially in the area of stellar clusters and magnetic activity. By the end of 2017, K2 
+will have observed more than 15 open clusters and associations, some on more than one occasion. The resulting data have 
+enabled ground-breaking research on a number of topics, including gyrochronology, stellar rotation and activity, planet 
+occurence rates, young stellar objects, and eclipsing binary systems. Astronomers working in these areas are warmly 
+invited to gather for discussion at BU this coming January.
+
+The motivations for this workshop include:
+
+* Summarizing the science done to date with K2 in the area of cool stars and open clusters
+* Forging new collaborations for future science with K2
+* Exploring synergies with other space missions (especially TESS), wavelength regions, and science areas.
+
+
+### Agenda
+
+This will be a small workshop with time for most participants to give a short overview of their research during the first 
+two days. (There will not be any posters.) We have reserved the final morning for "hack day" activities, including data how-to's 
+and high-level science product creation overviews. We will also encourage participants to spend time comparing technical 
+approaches to K2 data, including photometry.
+
+<h4 style="font-weight: bold;">Schedule TBA</h4>
+
+### Venue
+
+The workshop will take place on the Boston University campus.
+
+The nearest airport is the Boston Logan International Airport (BOS).
+We recommend using the MBTA system, which provides free public transportation from the airport.
+
+
+### Accommodation
+
+There is no on-site accommodation available for this meeting, but we can recommend the following nearby hotels:
+
+* <a href='https://cambridge.regency.hyatt.com/en/hotel/home.html'>The Hyatt Regency Cambridge</a> is just across
+the river and walkable (at most 15 minutes) and has rooms for approximately $150-200/night + tax.
+
+* <a href='http://www.bostonhotelbuckminster.com/'> Hotel Buckminster</a> is on the BU side of the river, in Kenmore Square.
+Prices range from $110-150/night + tax.
+
+* <a href='https://www.hotelcommonwealth.com/'> Hotel Commonwealth</a> is also in Kenmore Square, but a bit more pricey.
+
+* For other options, check out hotels on the C Green Line, such as the <a href='https://www.reservations.com/hotel/holiday-inn-boston-brookline?gclid=EAIaIQobChMIo7Hy6N3D1gIVir3tCh30-AbCEAAYAiAAEgJSyfD_BwE'>
+Holiday Inn Boston-Brookline</a> (about 20 minutes' ride).
+
+### Registration
+
+Registration will be available free of charge, apart from a small fee to be paid by cash or check on site. Please indicate 
```

Comment:
It can't be both free of charge and require a fee.  Let's delete the entire sentence for now until we are 100% sure whether we are charging or not.